<!-- Mission de suivi de waypoints -->
<launch>
	<arg name = "patx" default = "-18"/>
	<arg name = "paty" default = "-7"/>
	<arg name = "id" default = "200"/>
	<arg name = "patrad" default = "5"/>

	<node name="obstacle" pkg="process" type="behavior_client_exemple.py">
		<param name="x" value="$(arg patx)"/>
		<param name="y" value="$(arg paty)"/>
		<param name="behav_type" value="obst_point2"/>
		<param name="force" value="5"/>
		<param name="b_id" value="$(arg id)"/>
		<param name="radius" value="$(arg patrad)"/>

	</node>
	
</launch>
